btree/btree.c

```diff
@@ -2900,41 +3020,45 @@ static void ff_make(struct slot *slot)
 	int             total_val_size;
 
 	if (h->ff_used == 0 || slot->s_idx == h->ff_used) {
+		if (IS_EMBEDDED_INDIRECT(slot->s_node) && h->ff_used == 0)
+			ff_dir_init(slot->s_node);
```

Comment:
In ff_init(), we do not know the value size as it can be different for internal node or leaf node.
This will be done as a separate task by passing the level parameter to the ff_init()